### PR TITLE
Improve UG

### PR DESCRIPTION
**Pull request by seanpzk**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -900,7 +912,7 @@ Removes all your entries from EduTrack when you want to start fresh.
 
 ### Saving the data
 
-EduTrack's got your back. Your data is automatically saved on your hard disk after any command that changes the data, so no need to worry about saving manually.
+EduTrack got your back. Your data is automatically saved on your hard disk after any command that changes the data, so no need to worry about saving manually.
```
Comment (sudarshan2401) [inline docs/UserGuide.md]: I think with the ('s) is actually grammatically correct. If not, it can be replaced with "EduTrack has got your back"

**Review by sudarshan2401 — APPROVED** [review]
LGTM other than the possible grammatical issue!

**Review by DonovanJJ — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (seanpzk) [reply docs/UserGuide.md]: Ohh oops, thanks!!

**Pull request merged**